boards/arm/stm32_min_dev/doc/index.rst

```diff
@@ -108,6 +108,16 @@ The stm32_min_dev board configuration supports the following hardware features:
 
 Other hardware features are not supported by the Zephyr kernel.
 
+I2C
```

Comment:
Oh! Tbh, I wasn't aware other pin configuration were not documented, I requested this update as I usually  do for other boards. So things are not quite homogeneous now on this board doc, but nvm, let's keep it now. Thanks